downstream/titles/catalog/getting-started/master.adoc

```diff
@@ -225,11 +225,11 @@ Each approval process is assigned a sequence ID that determines the priority in
 You can edit the sequence number for an approval process from the Approval process list view.
 
 . Select *Approval* from the main navigation.
-. Click the *Approval processes* tab.
+. Click the btn:[Approval processes] tab.
```

Comment:
Approval processes is a tab, so it should be just bold with an * on either side.